indico/modules/events/registration/models/forms.py

```diff
@@ -136,6 +136,12 @@ class RegistrationForm(db.Model):
         nullable=False,
         default=False
     )
+    #: Whether number of registrations should be displayed
+    publish_number_of_registrations = db.Column(
```

Comment:
What about `publish_registration_count`? `of` in a variable name usually feels too verbose